.semaphore/semaphore.yml

```diff
@@ -12,28 +12,27 @@ blocks:
         commands:
           - echo "Installing Performous dependencies..."
           - sudo apt -y update
-          - sudo apt -y install cmake gettext help2man clang gcc-7 g++-7 libepoxy-dev libsdl2-dev libcairo2-dev libpango1.0-dev librsvg2-dev libboost-all-dev libavcodec-dev libavformat-dev libswscale-dev libswresample-dev libpng-dev libjpeg-dev libxml++2.6-dev portaudio19-dev libopencv-dev libportmidi-dev libqrencode-dev libicu-dev libcpprest-dev libglm-dev libopenblas-dev libfftw3-dev
+          - sudo add-apt-repository -y ppa:litenstein/performous-bionic
```

Comment:
For what would we need this ppa?

Reply:
As mentioned on discord we need it for Boost.ASIO which is in ubuntu 18.04 at 1.65 while we at least require boost 1.66

Ubuntu 19.04 is shipped with boost 1.67

Reply:
@performous/core-owners What do we think of using a personal more updated ppa?
I'm not quite fond of it myself but if it's necessaryit's necessary..
I'm not fond of it for the reason as it might mess up the users system since we're not OS maintainers we can't really tell the impact. And aside from this we also need to keep our documentation up to date regarding what and how to install stuff properly so you can build performous

Reply:
As I said in Discord; my issue is that despite being arguably the most popular Linux flavor, Ubuntu keeps some really outdated packages here and there and sometimes the reasons are dubious at best... for example, take LibXML++; there is no sane reason for the current cutting-edge of ubuntu still having only LibXML++2.6; and yet it does (seeing how, even if some packages would break by using 3.0; they could ship both, since they're designed to coexist).

With regards to boost, Back when 18.04 first came out, 1.65 was already almost a full year old, 1.66 was out and 1.67 came out around the same time as Ubuntu 18.04. I just don't see it as justifiable that in TWO-YEARS of their "LTS", they didn't update the goddamn boost version that was already obsolete when it first came out.

I think it's ridiculous to hold updating our code when even Windows has easily-accessible libraries that are kept more updated.

I might have missed something of course but from a bit of search into package-dependencies, the only thing that might be broken by adding the updated icu and boost packages (in addition to that ppa, they are also available on separate ppas to minimize new packages) to bionic is an old version of OpenTTD (another game)

Do keep in mind that packages in launchpad have to build before they are published, and if they won't build if they break anything important in the first place.